app/views/catalog/_facet_pagination.html.erb

```diff
@@ -1,3 +1,13 @@
+    <% if @facet.index_pagination && params["facet.sort"] == 'index' %>
+      <div class="alpha-filter">
+        <%= t('blacklight.search.facets.filter') %>:
+        <%= link_to(t('blacklight.search.facets.clear'), @pagination.params_for_resort_url('index', params.except(:"facet.prefix")), :data => {:ajax_modal => "preserve"}) %>
```

Comment:
Can you convert these to use Ruby 1.9-style hashes? While there are 1.8-style hashes around the project, we've been trying to migrate them to the new style as those lines of code are touched.